tutorials/introductory/getting_started.py

```diff
@@ -0,0 +1,767 @@
+# are responsible for plotting and configuring additional details. Each
+# Figure can contain multiple Axes, but each Axes is specific to one
+# Figure.
+#
+# In a Figure, each Axes can contain any number of visual elements. An Axes may
+# have more than one type of visualization of data. From the `Plotting`_
+# section above, the Axes in both explicit and implicit strategies contain
+# multiple types of visualizations of data on a single Axes. Each of these
+# types are specific to the Axes they are in.
+#
+# Other Components
+# ^^^^^^^^^^^^^^^^
+#
+# :class:`~matplotlib.artist.Artist`
+#
+# Artists are broad Matplotlib objects that display visuals. These are the
+# visible elements when the Figure is rendered. They correspond to a specific
+# Axes and cannot be shared or transferred. In Matplotlib programming, all
+# objects for display are Artists.
+#
+# .. note::
+#
+#   Axes and Axis are not synonymous. Axis refers to
+#   :class:`~matplotlib.axis.Axis`.
+#
+# Manipulating Artists
+# --------------------
+#
+# With simple plots, Matplotlib automatically generates the basic elements of
+# a graph. For more control over the visual, use Artists and its methods.
+#
+# Matplotlib generates additional visual elements as Artists in the form of
+# objects. As Artists, each has its own respective methods and functions.
+# Explicit and implicit approaches use different methods and are not
+# interchangeable.
+#
+# The table below compares common Artists and their different methods.
+#
+# +-----------------------+--------------------------+------------------------+
+# | Artist                | Explicit                 | Implicit               |
+# +=======================+==========================+========================+
+# | Visible elements from | Each specific Axes has   | The ``pyplot`` module  |
+# | rendered Figure.      | its own method for       | manages Artists based  |
+# |                       | artists.                 | on most recent Figure  |
+# |                       |                          | or Axes.               |
+# +-----------------------+--------------------------+------------------------+
+# | X-axis labels         | ``ax.set_xticks()``      | ``plt.xlabel()``       |
+# |                       | ``ax.set_xticklabels()`` |                        |
+# +-----------------------+--------------------------+------------------------+
+# | Y-axis labels         | ``ax.set_yticks()``      | ``plt.ylabel()``       |
+# |                       | ``ax.set_yticklabels()`` |                        |
+# +-----------------------+--------------------------+------------------------+
+# | Title (Axes)          | ``ax.set_title()``       | ``plt.title()``        |
+# +-----------------------+--------------------------+------------------------+
+# | Legend (Axes)         | ``ax.legend()``          | ``plt.legend()``       |
+# +-----------------------+--------------------------+------------------------+
```

Comment:
maybe pull this out into a different table w/ the other "we are drawing a thing" artist like lines and scatter - @timhoffm makes the adding something new vs/ configuring a part of the axes/plot that exists

Reply:
Open to discuss current appearance during call!